src/j1cli.js

```diff
@@ -199,7 +197,8 @@ async function initializeJ1Client() {
     password: program.password,
     poolId: J1_USER_POOL_ID,
     clientId: J1_CLIENT_ID,
-    accessToken: program.key || J1_API_TOKEN
+    accessToken: program.key || J1_API_TOKEN,
+    dev: true
```

Comment:
Remove this. We should add configurability around enabling dev mode via an environment variable later.

Reply:
Got it.